src/lib/directive.js

```diff
@@ -59,7 +59,8 @@ const _ = require('lodash');
                         target.displayName ||
                         target.firstname + ' ' + target.lastname ||
                         target.preferredEmail,
-                  email: target.email || target.preferredEmail
+                  email: target.email || target.preferredEmail,
+                  subject: attrs.opInboxComposeSubject
```

Comment:
```suggestion
                  subject: attrs.opInboxComposeSubject || ''
```

probably should have a default value to avoid having `subject=undefined` in the URL.

Reply:
Good remark but maybe we could have a better default than an empty string - or avoid passing the subject in the URL if we have none.

Reply:
```
if (attrs.opInboxComposeSubject) {
  targetToAdded.subject = attrs.opInboxComposeSubject;
}
```